python/nav/statemon/db.py

```diff
@@ -113,16 +116,17 @@ def cursor(self):
                 cursor.execute('SELECT 1')
             except psycopg2.InternalError as err:
                 if err.pgcode == IN_FAILED_SQL_TRANSACTION:
-                    debug("Rolling back aborted transaction...", 2)
+                    LOGGER.critical("Rolling back aborted transaction...")
                     self.db.rollback()
                 else:
-                    debug("PostgreSQL reported an internal error I don't know "
-                          "how to handle: %s (code=%s)" % (
-                          pg_err_lookup(err.pgcode), err.pgcode), 2)
+                    LOGGER.critical("PostgreSQL reported an internal error "
+                                    "I don't know how to handle: %s "
+                                    "(code=%s)", pg_err_lookup(err.pgcode),
+                                    err.pgcode)
                     raise
         except Exception as err:
-            debug(str(err), 2)
-            debug("Could not get cursor. Trying to reconnect...", 2)
+            LOGGER.critical(str(err))
+            LOGGER.critical("Could not get cursor. Trying to reconnect...")
```

Comment:
Interesting, `critical()` is only used in `nav.statemon.*`. `exception()` is in use several places so at least in the case above, when logging the actual error-message, the most proper fix is using `exception()`.